src/python/pybind11/src/pyxrt.cpp

```diff
@@ -358,4 +361,131 @@ PYBIND11_MODULE(pyxrt, m) {
+            a = a | b;
+            return a;
+        })
+        .def("__iand__", [](xrt::ext::bo::access_mode &a, xrt::ext::bo::access_mode b) {
+            a = a & b;
+            return a;
+        })
+        ;
+    
+    pyextbo
+        .def(py::init([](const xrt::device& device, void* userptr, size_t sz, xrt::ext::bo::access_mode access) {
+            return new xrt::ext::bo(device, userptr, sz, access);
+        }))
+        .def(py::init([](const xrt::device& device, void* userptr, size_t sz){
+            return new xrt::ext::bo(device, userptr, sz);
+        }))
+        .def(py::init([](const xrt::device& device, size_t sz, xrt::ext::bo::access_mode access) {
+            return new xrt::ext::bo(device, sz, access);
+        }))
+        .def(py::init([](const xrt::device& device, size_t sz) {
+            return new xrt::ext::bo(device, sz);
+        }))
+        .def(py::init([](const xrt::device& device, xrt::pid_type pid, xrt::bo::export_handle ehdl) {
+            return new xrt::ext::bo(device, pid, ehdl);
+        }))
+        .def(py::init([](const xrt::hw_context& hwctx, size_t sz, xrt::ext::bo::access_mode access) {
+            return new xrt::ext::bo(hwctx, sz, access);
+        }))
+        .def(py::init([](const xrt::hw_context& hwctx, size_t sz) {
+            return new xrt::ext::bo(hwctx, sz);
+        }))
+        .def(py::init([](const xrt::hw_context& hwctx, xrt::pid_type pid, xrt::bo::export_handle ehdl) {
+            return new xrt::ext::bo(hwctx, pid, ehdl);
+        }))
+        ;
+    
+    py::class_<xrt::ext::kernel, xrt::kernel> pyextkernel(ext, "kernel", "Represents an external kernel object");
+
+    pyextkernel
+        .def(py::init([](const xrt::hw_context& ctx, const xrt::module& mod, const std::string& name) {
+            return new xrt::ext::kernel(ctx, mod, name);
+        }))
+        .def(py::init([](const xrt::hw_context& ctx, const std::string& name) {
+            return new xrt::ext::kernel(ctx, name);
+        }))
+        ;
+
+    /*
+    *
+    * xrt::runlist
+    * 
+    */
+
+    py::class_<xrt::runlist> pyrunlist(m, "runlist", "Represents a list of runs to be executed");
+    
+    pyrunlist
+        .def(py::init([](){
+            return new xrt::runlist();
+        }))
+        .def(py::init([](const xrt::hw_context& hwctx) {
+            return new xrt::runlist(hwctx);
+        }))
+        .def("add", ([](xrt::runlist &r, const xrt::run& run) {
+            r.add(run);
+        }), "Add a run to the runlist")
+        .def("execute", ([](xrt::runlist &r) {
+            r.execute();
+        }), "Execute all runs in the runlist")
+        .def("wait", ([](xrt::runlist &r) {
+            r.wait();
+        }), "Wait for all runs in the runlist to complete")
+        .def("wait", ([](xrt::runlist &r, const std::chrono::milliseconds& timeout) {
+            return r.wait(timeout);
+        }), "Another way to wait for all runs in hte runlist to complete");
```

Comment:
Update the documentation "Wait for the specified timeout for the runlist to complete".